src/http.rs

```diff
@@ -1,16 +1,24 @@
 // Copyright 2020 Cognite AS
 //! The HTTP Layer
 
-use http_types::headers;
+#[cfg(feature = "surf")]
+use surf::{Client, http::headers::HeaderName};
+#[cfg(feature = "surf")]
+pub use surf::{Error, RequestBuilder, Response};
```

Comment:
`Response` was exposed as `crate::client` needs it, and thinking about how these public types are supposed to be used, I think that makes sense to also expose it publicly.

Reply:
Hmm, I'll need to look back at this - Its not really ideal that we leak these details.

Reply:
I've removed all the exposure of that, took a new approach to this feature - please see #38.